usl_pipeline/usl_lib/usl_lib/storage/cloud_storage.py

```diff
@@ -13,3 +13,6 @@
 FLOOD_SIMULATION_CONFIG_BUCKET = (
     os.environ.get("BUCKET_PREFIX", "") + "climateiq-flood-simulation-config"
 )
+LABEL_CHUNKS_BUCKET = (
+    os.environ.get("BUCKET_PREFIX", "") + "climateiq-study-area-label-chunks"
```

Comment:
If we ran this locally, will the bucket prefix be "test-"?

Reply:
It would only be 'test-' if you've set the BUCKET_PREFIX environmental variable to 'test-'